src/components/HeaderMenu/HeaderMenu.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { Layout, Menu, Typography, Button, theme } from 'antd';
+import { ItemType } from 'antd/lib/menu/hooks/useItems';
+import { NavLink } from 'react-router-dom';
+import styles from './HeaderMenu.module.scss';
+import routePaths from '../../utils/constants/routePaths';
+import useSelectedNavMenuKeys from './useSelectedNavMenuKeys';
+import colorTheme from '../../store/theme';
```

Comment:
You need to select a pattern of import for your project. It will be more readable.
For example: first, we import js libs from modules, then our js files, then store files, and at the end, we can import styles. Also, we can use an empty line to separate them.
And after that use this pattern everywhere.